---
layout: standard
title: PCR
---

{% include protocols_header.html %}

<h1>PCR (Polymerase Chain Reaction)</h1>
<h2>Rationale</h2>
<p>
    PCR is a method used to amplify specific sections of a DNA sequence.
    It uses a heat-resistant DNA polymerase (DNAP), a pair of DNA oligonucleotides (primers) to define the amplicon and prime the reaction,
    and dNTPs (deoxyribonucleoside triphosphates: A, T, C, G) as substrates for the amplicon elongation.
</p>
<p>
    The reaction needs to cycle through three steps to copy a single DNA strand once:
    <ol>
        <li>Denaturation: heat to 95~98°C to de-hybridize the template DNA strand</li>
        <li>Annealing: cool down to the primers' melting temperature (Tm) to bind them to the template</li>
        <li>Elongation: heat to 72°C to allow elongation by the DNAP</li>
    </ol>
    The quick changes in temperature are made inside a thermocycler,
    and this normally is done 25~30 times with an initial longer denaturing step and a final longer elongation step.
    The products of a cycle serve as template for the next, ensuring geometric growth of the number of template DNA copies until dNTPs are depleted.
</p>

<h2>Q5 Polymerase protocol</h2>

<p>Q5 is a proprietary (NEB) high-fidelity DNAP used mostly for molecular cloning purposes.</p>
<p>Check the Tm for annealing regions on Q5 reactions in <a href="https://tmcalculator.neb.com/#!/main">their website</a>.</p>
<p>Adapted from <a href="https://www.neb.com/en/protocols/2013/12/13/pcr-using-q5-high-fidelity-dna-polymerase-m0491">NEB's protocol</a>, assemble the following reaction:</p>
<table>
    <tr>
        <th>Component</th>
        <th>10uL reaction</th>
        <th>25uL reaction</th>
        <th>Final conc.</th>
    </tr>
    <tr>
        <td>-</td>
        <td>-</td>
        <td>-</td>
        <td>-</td>
    </tr>
    <tr>
        <td>10 mM dNTPs</td>
        <td>1 uL</td>
        <td>2.5 uL</td>
        <td>100 uM</td>
    </tr>
    <tr>
        <td>10 uM primers </td>
        <td>0.5 uL (each)</td>
        <td>1.25 uL (each)</td>
        <td>0.5 uM</td>
    </tr>
    <tr>
        <td>5X Q5 buffer</td>
        <td>2 uL</td>
        <td>5 uL</td>
        <td>1X</td>
    </tr>
    <tr>
        <td>Q5 polymerase</td>
        <td>0.1 uL</td>
        <td>0.25 uL</td>
        <td>0.02 U/uL</td>
    </tr>
    <tr>
        <td>Water</td>
        <td>6 uL</td>
        <td>15 uL</td>
        <td>-</td>
    </tr>
</table>

<p>
    Also adapting NEB's protocol: set up a thermocycler with the following program:
</p>

<table>
    <tr>
        <th>Step</th>
        <th>Instruction</th>
        <th>Time</th>
    </tr>
    <tr>
        <td>1</td>
        <td>98°C</td>
        <td>5 min</td>
    </tr>
    <tr>
        <td>2</td>
        <td>98°C</td>
        <td>30 s</td>
    </tr>
    <tr>
        <td>3</td>
        <td>anneal temp.</td>
        <td>30 s</td>
    </tr>
    <tr>
        <td>4</td>
        <td>72°C</td>
        <td>30 s /amplicon kb</td>
    </tr>
    <tr>
        <td>5</td>
        <td>Back to step 2</td>
        <td>25~30 times</td>
    </tr>
    <tr>
        <td>6</td>
        <td>72°C</td>
        <td>5 min</td>
    </tr>
    <tr>
        <td>7</td>
        <td>4°C</td>
        <td>-</td>
    </tr>
</table>

<h2>Primer Design</h2>

<p>
    There are some general best-practices when designing primers for PCR:
    <ul>
        <li>Avoid primer dimers and secondary structure. Use Thermo's <a href="https://www.thermofisher.com/uk/en/home/brands/thermo-scientific/molecular-biology/molecular-biology-learning-center/molecular-biology-resource-library/thermo-scientific-web-tools/multiple-primer-analyzer.html">Multiple Primer Analyzer</a> for that.</li>
        <li>Aim for annealing regions around 20bp, with Tm around 60°C.</li>
        <li>Try to have the 3'-end of primers with a C or G.</li>
        <li>If using Benchling, you can click the proposed Tm it gives you at the bottom and change parameters like concentrations of primers, dNTPs and Mg.</li>
    </ul>
</p>

<h2>Tips</h2>
<ul>
    <li>
        When using quick denaturing and annealing steps (~15s) it's useful to pay attention to the ramping times in your thermocycler.
        If it considers temperature ramping into the cycle time, you might not stay at the desired temperature for long enough.
    </li>
    <li>
        Template complexity matters a lot for appropriate PCR efficiency. Genomic templates might require extra time in all steps.
    </li>
    <li>
        You can often run the reaction with half the amount of polymerase as recommended.
    </li>
    <li>
        Assembling the reaction is normally done at room temperature. To minimize primer dimer formation, some protocols recommend assembling the reaction at 4°C
        then transferring the tubes to a pre-heated thermocycler at 98°C. This attempts to stop the enzymes from extending the dimers before the first denaturation.
    </li>
</ul>